google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -558,6 +594,9 @@ private void replenishPool() {
   @GuardedBy("lock")
   private int maxSessionsInUse = 0;
 
+  @GuardedBy("lock")
+  private Map<String, PooledSession> allSessions = new HashMap<>();
```

Comment:
This is consistent with other fields in this class. I will fix them all separately.